app/interactors/access_limit/update_interactor.rb

```diff
@@ -24,12 +26,12 @@ def find_and_lock_edition
   def update_edition
     limit_type = params.require(:limit_type)
 
-    if limit_type == "none"
-      context.fail! if edition.access_limit.nil?
-      edition.remove_access_limit(user).save!
-    else
+    if LIMIT_TYPES.include?(limit_type)
       context.fail! if edition.access_limit&.limit_type == params[:limit_type]
```

Comment:
Shall we use the limit_type variable we assigned here too?

Reply:
Oh yes - thanks for spotting :-).